docker-compose.yml

```diff
@@ -2,32 +2,35 @@ version: "3.8"
 
 services:
   web:
-    image: "${DHIS2_IMAGE:-dhis2/core-dev:local}"
+    image: "${DHIS2_IMAGE:-dhis2/core}"
+    restart: unless-stopped
     ports:
-      - 127.0.0.1:8080:8080 # DHIS2
-      - 127.0.0.1:8081:8081 # Debugger: connect using commandline flag -agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=*:8081
-      - 127.0.0.1:9010:9010 # JMX port (for example for VisualVM)
+      - 127.0.0.1:${DHIS2_PORT:-8080}:8080 
     volumes:
+      - files:/opt/dhis2
       - ./docker/dhis.conf:/opt/dhis2/dhis.conf:ro
       - ./docker/log4j2.xml:/opt/dhis2/log4j2.xml:ro
     environment:
-      JAVA_OPTS: "-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=*:8081 \
-              -Dlog4j2.configurationFile=/opt/dhis2/log4j2.xml
-              -Dcom.sun.management.jmxremote \
-              -Dcom.sun.management.jmxremote.port=9010 \
-              -Dcom.sun.management.jmxremote.local.only=false \
-              -Dcom.sun.management.jmxremote.authenticate=false \
-              -Dcom.sun.management.jmxremote.ssl=false"
+      JAVA_OPTS: "-Dlog4j2.configurationFile=/opt/dhis2/log4j2.xml"
     depends_on:
       db:
         condition: service_healthy
+    healthcheck:
+      test: ["CMD-SHELL", "wget -q -S -O /dev/null http://127.0.0.1:${DHIS2_PORT:-8080}/dhis-web-commons/security/login.action 2>&1| grep -q 'HTTP.*200'"]
+      start_period: 120s
+      interval: 10s
+      timeout: 5s
+      retries: 5
+    security_opt:
+      - no-new-privileges:true
 
   db:
-    image: ghcr.io/baosystems/postgis:12-3.3
+    image: postgis/postgis
```

Comment:
Why use this image?

I would recommend [this](https://hub.docker.com/r/bitnami/postgresql) one which is also what we're using on Kubernetes.

Regardless of which image we're using. A tag should be present so we're not just using latest.

Reply:
Just a quick note here - the `postgis/postgis` image won't suffice, as it's only built for `linux/amd64` and we want this Docker compose file to also be usable by developers with the new Mac M* chips. 

The `ghcr.io/baosystems/postgis` one is built for `linux/arm64` as well and it provides some convenience, as we don't need to install `postgis` separately (like we do [here for the dhis2-db stack in the IM](https://github.com/dhis2-sre/im-manager/blob/master/stacks/dhis2-db/seed.sh#L19)), but it might actually be better to streamline this for consistency.  👍

Reply:
@michael-markevich Right, I should have been a bit more clear - the `postgis/postgis` image won't suffice for the developer docker compose file.

Still, it might be worth having the image we recommend for production use be the same one we use in the Instance Manager, as it has been tested the most. Technically, `postgis/postgis` is also a custom image, the "official" one is https://hub.docker.com/_/postgres.